src/main/java/kafdrop/service/KafkaHighLevelConsumer.java

```diff
@@ -187,9 +187,8 @@ private void initializeClient() {
             rec.timestampType(),
             0L,
             rec.serializedKeySize(),
-            rec.serializedValueSize(),
-            rec.key(),
-            deserialize(deserializer, rec.value()),
+            rec.serializedValueSize(), deserialize(deserializers.getKeyDeserializer(), rec.key()),
```

Comment:
Would you mind having each arg on its own line, otherwise it gets hard to read with so many arguments.

Reply:
Agree. Resolved in latest push

Reply:
Thanks @nicklester